microsoft-edge/extensions-chromium/getting-started/extension-sideloading.md

```diff
@@ -1,24 +1,28 @@
 ---
 title: Sideload an extension
-description: How to test an extension by sideloading it in the browser.
+description: Test an extension locally by sideloading it, so that it's locally installed in your instance of Microsoft Edge.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: extensions
-ms.date: 11/07/2022
+ms.date: 09/30/2024
 ---
 # Sideload an extension
 
-During development, you can use Microsoft Edge browser to run and debug your extension safely.  By sideloading your extension locally in your browser, you can run and test your extension without changing any settings. This helps you test new extensions safely.
+To test an extension quickly and safely, sideload the extension in your own copy of Microsoft Edge, which means locally installing the extension.  You can then run and test your extension without changing any settings.<!-- todo: what sort of settings are meant?  what is the contrasted approach? -->
```

Comment:
"sideloading" scenario is still unclear.  to clarify, state the installation approach that is contrasted with sideloading.

what sort of settings are meant? what is the contrasted approach?

@bhuvanapriyap

Reply:
resolved; clarified intro, added first h2 heading to clarify; clarified later h2 headings.  clarified title of article.
result, eg:
```
# Sideload an extension to install and test it locally

To test an extension quickly and safely, sideload the extension in your own copy 
of Microsoft Edge, which means locally installing the extension.  You can then run 
and test your extension without having to upload the extension to the Microsoft 
Edge Add-ins website (the _store_) and then install the extension from there.


<!-- ====================================================================== -->
## Locally installing and running an extension

To sideload an extension into Microsoft Edge, so that it's locally installed in your 
browser for testing:
```